src/DynamoCoreWpf/ViewModels/Search/SearchViewModel.cs

```diff
@@ -153,7 +153,7 @@ public IEnumerable<NodeSearchElementViewModel> FilteredResults
             }
             set
             {
-                filteredResults = value;
+                filteredResults = ToggleSelect(value);
```

Comment:
Can you write a unit test to make sure the first item is selected? also, a test to ensure if there no items in filteredResults .